### ISSUE TITLE
[1.16.3] Server crash

### ISSUE THREAD

**Issue by WhitePhant0m** [Ipsis/Woot#459]
[Crash Log](https://paste.gg/p/wphant0m/b990f3508fa74dacb59d7a3f48a808fd)

Player said that they had some Ice & Fire mobs being spawned by their woot factory, not sure which of them is causing this.

woot-1.16.3-1.0.0.0
forge 34.1.25

Edit: Update to the issue, seems to be all of myrmex from Ice & Fire

**Comment by Ipsis** [comment]
As a stop-gap until I work out what is going on with the Myrmex mobs (looks like I am failing to get the health of the mob from Ice and Fire - I think), then you can blacklist the particular mob(s) or the complete mod in the config file.

Complete mod
`[policy.blacklist]
captureMod = ["botania", "iceandfire"]`

Specific mobs
`[policy.blacklist]
captureEntity: [ "iceandfire:xxx". "iceandfire:xxx2" ]`

**Comment by Ipsis** [comment]
I've added an extra check health lookup failures which don't seem to work with the Myremex mobs. This is available in 1.0.0.2.

**Comment by Ipsis** [comment]
Finally tracked down what was going on with Ice & Fire.
The Myrmex mobs cause an exception when EntityType.loadEntityAndExecute is called.

I'll test the rest of the mobs in Ice & Fire, but I think it is just the Myrmex that are the issue, so I'll internally blacklist them.

```
Caused by: java.lang.NullPointerException
	at net.minecraft.nbt.NBTUtil.func_186860_b(SourceFile:167) ~[?:?]
	at net.minecraft.nbt.CompoundNBT.func_186857_a(CompoundNBT.java:133) ~[?:?]
	at com.github.alexthe666.iceandfire.entity.EntityMyrmexBase.func_70037_a(EntityMyrmexBase.java:290) ~[?:2.1.3-1.16.4]
	at com.github.alexthe666.iceandfire.entity.EntityMyrmexQueen.func_70037_a(EntityMyrmexQueen.java:117) ~[?:2.1.3-1.16.4]
	at net.minecraft.entity.Entity.func_70020_e(Entity.java:1495) ~[?:?]
```
